src/components/BookProgress.css

```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap');
 
 .book-progress {
   display: flex;

```

Comment:
**Changes Required**
Good effort so far, bravo. However;
- [ ] The CSS selector property `display` with the value of `flex` is repeated multiple times. You want to follow best CSS practice of **Don't Repeat Yourself. (DRY).** Please let us refactor it by creating for instance a `.display-flex` in your `.jsx` file and having a `.display-flex { display: flex}` in your CSS file. Please do this for all repeated Selector properties i.e `(color, font-size, etc)` in all your CSS files.

Reply:
Overflow hidden added to cater for small laptop screens 
Display flex repeats Sorted